README.md

```diff
@@ -140,26 +140,26 @@ Example filled out table (note how I also moved around and deleted some rows):
 ```
 raw_name	HMM.Family	HMM_length	notes
 rpoB_13	rpoB	2842	This is the taxonomic marker gene I plan to use
-cyc2_0	cyc2_3GSB	411	
-dsrA_4	dsrA	369	
-soxB_14	soxB	570	
-BChl_A_8	fmoA	369	
-TIGR01115_9	pufM	570	
-TIGR01281_10	bchL	369	
-TIGR02019_11	bchJ	570	
+cyc2_0	cyc2_3GSB	411
```

Comment:
@NickiShaw These trailing tabs are needed for the TSV file to be complete -- can you please re-add them?